js/background.js

```diff
@@ -300,6 +300,12 @@
       storage.put('primaryDevicePubKey', textsecure.storage.user.getNumber());
     }
 
+    // 4th August 2020 - Force wipe of secondary devices as multi device is being disabled.
+    if (storage.get('isSecondaryDevice')) {
+      await window.deleteAccount();
```

Comment:
should we make a toast, wait 10sec and then delete?

Reply:
I think we should just delete straight away since we've given a warning to the users beforehand.